pages/main_page.py

```diff
@@ -0,0 +1,104 @@
+        amount_of_cards_with_picture = len(picture_on_card)
+        print('Amount of cards with pictures ' + str(amount_of_cards_with_picture))
+        i = 0
+        list_of_pictures = []
+        for element in picture_on_card:
+            new_picture = element.get_attribute("src")
+            print('Picture # ' + str(i) + '\n' + new_picture)
+            list_of_pictures.append(new_picture)
+            i = i + 1
+        list_of_pictures_set = set(list_of_pictures)
+        print('Total ' + str(len(list_of_pictures)) + ' pictures, ' + str(len(list_of_pictures_set)) + ' of which are unique')
+        assert len(list_of_pictures) == len(list_of_pictures_set), 'Duplicate pictures'
+
+    def cards_have_different_content(self):
+        content_on_card = self.browser.find_elements(*MainPageLocators.CONTENT_ON_CARD)
+        amount_of_cards_with_content = len(content_on_card)
+        print('Amount of cards with content ' + str(amount_of_cards_with_content))
+        i = 0
+        list_of_content = []
+        for element in content_on_card:
+            new_content = element.text
+            list_of_content.append(new_content)
+            print('Content # ' + str(i) + '\n' + new_content)
+            i = i + 1
+        list_of_content_set = set(list_of_content)
+        print('Total ' + str(len(list_of_content)) + ' contents, ' + str(len(list_of_content_set)) + ' of which are unique')
+        assert len(list_of_content) == len(list_of_content_set), 'Duplicate content'
+
+    def cards_have_different_links(self):
+        link_on_card = self.browser.find_elements(*MainPageLocators.LINK_ON_CARD)
+        amount_of_cards_with_link = len(link_on_card)
+        print('Amount of cards with link ' + str(amount_of_cards_with_link))
+        i = 0
+        list_of_links = []
+        for element in link_on_card:
+            new_link = element.get_attribute('href')
+            list_of_links.append(new_link)
+            print('Link # ' + str(i) + '\n' + new_link)
+            i = i + 1 
+        list_of_links_set = set(list_of_links)
+        print('Total ' + str(len(list_of_links)) + ' links, ' + str(len(list_of_links_set)) + ' of which are unique')
+        assert len(list_of_links) == len(list_of_links_set), 'Links do not lead to internal pages'
+
+class DeparturePage(BasePage):
+    def click_on_the_departure_link(self):
+        menu_item = self.browser.find_element(*MainPageLocators.MENU_ITEMS)
+        menu_item.click()
+
+    def should_be_departure_page(self):
+        departure_header = self.is_element_present(*DeparturePageLocators.DEPARTURE_HEADER)
+        assert departure_header, 'No go to departure page'
+        current_url_departure = self.browser.current_url
+        print('Current url departure',current_url_departure)
+        assert 'departure' in current_url_departure, 'The link does not lead to the departure page ' + current_url_departure
```

Comment:
assert 'departure' in current_url, f"This is not departure page, expected 'departure' in the {current_url_departure}"